.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
While you're in this file, can you add what we have in Telescope? https://github.com/Seneca-CDOT/telescope/blob/master/.vscode/settings.json